src/sidebar/components/SearchInput.tsx

```diff
@@ -193,18 +193,9 @@ export default function SearchInput({
           <IconButton
             icon={SearchIcon}
             onClick={() => input.current?.focus()}
-            size="xs"
+            size="custom"
             title="Search annotations"
-            // The containing form has a white background. The top bar is only
-            // 40px high. If we allow standard touch-minimum height here (44px),
-            // the visible white background exceeds the height of the top bar in
-            // touch contexts. Disable touch sizing, then add back the width
-            // rule to keep horizontal spacing consistent.
-            // FIXME:
-            // Button/IconButton are due a pass for responsive styling and
-            // appropriate styling API controls (@hypothesis/frontend-shared)
-            disableTouchSizing
-            classes="touch:min-w-touch-minimum"
+            classes="touch:min-w-touch-minimum p-1"
```

Comment:
Setting `size="custom"` disables the classes that used to be disabled by `disableTouchSizing`.

Then we just need to add `p-1`, which is the equivalent to `size="xs"`.

Reply:
Just realized that comment (or part of it) should be kept to explain why `size="custom"` is used.